packages/scrubber-bar/xyz.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
curious on this addition?  i guess the yarn start wanted it?

Reply:
so sorry same problem 😭  -removing